### ISSUE TITLE
Circular dependencies (prevents proper `rollup`)

### ISSUE THREAD

**Issue by danielgindi** [Marak/colors.js#233]
Rollup spits those out:
```
Circular dependency: node_modules\colors\lib\colors.js -> node_modules\colors\lib\maps\america.js -> node_modules\colors\lib\colors.js
Circular dependency: node_modules\colors\lib\colors.js -> node_modules\colors\lib\maps\america.js ->  commonjs-proxy:C:\Users\Daniel.SILVERBOLT\Desktop\git\endpoint-analyzer\node_modules\colors\lib\colors.js -> node_modules\colors\lib\colors.js
Circular dependency: node_modules\colors\lib\colors.js -> node_modules\colors\lib\maps\zebra.js -> node_modules\colors\lib\colors.js
Circular dependency: node_modules\colors\lib\colors.js -> node_modules\colors\lib\maps\rainbow.js -> node_modules\colors\lib\colors.js
Circular dependency: node_modules\colors\lib\colors.js -> node_modules\colors\lib\maps\random.js -> node_modules\colors\lib\colors.js
```

It is a huge problem, and will make code unexecutable after "rolling up".
The rollup process will convert commonjs deps into "static imports", which do not support circular dependencies.
Then it will end up with a flat hierarchy where there's a reference to a (yet) undefined variable.

**Comment by DABH** [comment]
This should be fixed in colors@1.3.1.  Let us know if the issues persist.  Thanks!

**Comment by danielgindi** [comment]
Amazing, thanks :-)
